WeakAurasOptions/RegionOptions/Icon.lua

```diff
@@ -90,7 +90,6 @@ local function createOptions(id, data)
       name = L["Glow Type"],
       order = 21,
       values = WeakAuras.glow_types,
-      hidden = function() return not data.glow end,
```

Comment:
What about glow color ?

Reply:
I didn't include it because it wasn't part of the original ticket. Do you want me to include that anyways?

Reply:
Well, I think the same problem applies to it too.